styles.css

```diff
@@ -127,4 +127,211 @@
     cursor: pointer;
     color: #6070ff;
   }
+
+  .canopy {
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    align-items: center;
+    width: 100%;
+    gap: 1rem;
+    margin: 1rem 0;
+  }
+
+  .canopy img {
+    border-radius: 100%;
+  }
+
+  .card h1 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .button {
+    padding: 12px;
+    gap: 10px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #6070ff;
+  }
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    gap: 2rem;
+    margin-top: 7.125rem;
+    padding: 1rem 2rem;
+  }
```

Comment:
- [ ] I think it would be better if you used the ``grid`` layout rather than the ``flex`` box, because it was one of the requirements for your projects 

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.